### PR TITLE
jbproject payer deployer

### PR DESCRIPTION
**Pull request by mejango**
convenience contract to deploy project payers.

### CODE REVIEW

File: contracts/JBETHERC20ProjectPayer.sol [hunk 1]
```diff
@@ -218,15 +217,19 @@ contract JBETHERC20ProjectPayer is IJBProjectPayer, Ownable {
...
+        _amount, // ignored if the token is JBTokens.ETH.
+        _projectId,
+        _beneficiary,
+        _minReturnedTokens,
+        _preferClaimedTokens,
+        _memo,
+        _metadata
+      );
+      // Otherwise just add to balance so tokens don't get issued.
+    else _terminal.addToBalanceOf{value: _payableValue}(_projectId, _amount, _memo);
```
Comment (drgorillamd) [inline contracts/JBETHERC20ProjectPayer.sol:233]: dis?

File: contracts/JBETHERC20ProjectPayer.sol [hunk 2]
```diff
@@ -218,15 +217,19 @@ contract JBETHERC20ProjectPayer is IJBProjectPayer, Ownable {
...
+    // Pay if there's a beneficiary to receive tokens.
+    if (_beneficiary != address(0))
+      // Send funds to the terminal.
+      _terminal.pay{value: _payableValue}(
+        _amount, // ignored if the token is JBTokens.ETH.
+        _projectId,
+        _beneficiary,
+        _minReturnedTokens,
+        _preferClaimedTokens,
+        _memo,
```
Comment (drgorillamd) [inline contracts/JBETHERC20ProjectPayer.sol:229]: calldata (same as in terminal.pay),

Comment (mejango) [reply contracts/JBETHERC20ProjectPayer.sol:229]: making it calldata in _pay makes it weird calling from the `receive()` fn 
image

Comment (mejango) [reply contracts/JBETHERC20ProjectPayer.sol:229]: changed them in `pay` tho ayyayaya

**Pull request merged**